content/proposals.md

```diff
@@ -79,6 +79,19 @@ Please [see
 MSC1779](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/1779-open-governance.md)
 for full details of the project's Guiding Principles.
 
+For the purposes of transparency and to underpin the "greater benefit"
+principle, proposals MUST disclose possible conflicts of interest where
+they might reasonably exist. Authors MAY do this by filling out the
+following checklist and publishing it together with their proposal.
+
+```
+- [ ] I am being paid for working on this proposal by <ORGANIZATION>
```

Comment:
```suggestion
- [ ] I am being paid (directly or indirectly) for working on this proposal by <ORGANIZATION>
```
@Gnuxie would that be better though?

Reply:
yeah that makes sense :+1:

Reply:
What I was trying to get at in the other thread is that only direct relationships should have to be disclosed. I think a lot of times, people writing MSCs will probably not even know what business relationships their employer has with other companies. Empirically, the joint ventures in Matrix projects can be quite complex. This formulation now makes me worry about having to name every other organisation that my employer gets income from.

Reply:
Well it does mean that the disclosure process is quite weak and doesn't put any accountability onto matrix vendors. All the disclosure is for individuals and allows vendors to hide behind their employees who will obviously only be doing work for their employer... which we likely already knew. I'm not expecting anyone to come up with a magic solution to fix that but it kinda sucks a bit now that i realise it.

Reply:
It is a double standard though.. really if your employer wants to contribute to matrix then they're gonna have to be upfront and make a disclosure statement on the MSC... if that's too much or legally grey for them... then why's the same not apply to individuals?

Reply:
To illustrate why this is a big deal: It allows the foundation to claim that the spec process has a disclosure process which on the surface sounds good. And does apply to individuals. But it does not apply to the interests of any matrix vendor. Because their disclosure is as simple as "I am a vendor employee", and there is no disclosure.

Reply:
Whereas as an individual i would likely need to declare all of the following information:
- That i have an option contact to buy shares as part of my previous employment at vccorp.example.com
- I am receiving funds from a grant from nice-people.example.com to complete a feature that is relevant to the MSC. <link to grant details>
- I am carrying out contractual work for small-company.example.com to complete xyz feature

This is a lot of relevant information that i should disclose. But the moment i create a legal company then i don't need to disclose anything.

Reply:
Yeah, this is definitely an issue. 😕

I'm not sure how to fix this. In a way, the entire disclosure builds on good faith and we cannot systematically verify it. The only thing we can do is, if at some point it comes out that you have misrepresented your conflicts of interest,  "you then get shunned as a bad faith actor" (as Rich put it on [#1700](https://github.com/matrix-org/matrix-spec/issues/1700#issuecomment-2674076034)).

Therefore, I was trying to phrase the checklist in a way that doesn't make the good faith, normal people obsess about their disclosures given that we have limited control over the bad faith people anyway.